ci/deploy_app.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+REGION="${REGION:=europe-west2}"
+CPU="${CPU:=4}"
+MEMORY="${MEMORY:=4G}"
+CONCURRENCY="${CONCURRENCY:=80}"
+CDN_URL="${CDN_URL:=https://cdn.census.gov.uk}"
+NEW_RELIC_APP_NAME="${NEW_RELIC_APP_NAME:=}"
+NEW_RELIC_LICENSE_KEY="${NEW_RELIC_LICENSE_KEY:=}"
+COOKIE_SETTINGS_URL="${COOKIE_SETTINGS_URL:=#}"
+ADDRESS_LOOKUP_API_URL="${ADDRESS_LOOKUP_API_URL:=#}"
+ADDRESS_LOOKUP_API_AUTH_ENABLED="${ADDRESS_LOOKUP_API_AUTH_ENABLED:=False}"
+ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS="${ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS:=10}"
+WEB_SERVER_TYPE="${WEB_SERVER_TYPE:=gunicorn-threads}"
+WEB_SERVER_WORKERS="${WEB_SERVER_WORKERS:=7}"
+WEB_SERVER_THREADS="${WEB_SERVER_THREADS:=10}"
+WEB_SERVER_UWSGI_ASYNC_CORES="${WEB_SERVER_UWSGI_ASYNC_CORES:=10}"
+DATASTORE_USE_GRPC="${DATASTORE_USE_GRPC:=True}"
+MIN_INSTANCES="${MIN_INSTANCES:=1}"
+MAX_INSTANCES="${MAX_INSTANCES:=1}"
+EQ_NEW_RELIC_ENABLED="${EQ_NEW_RELIC_ENABLED:=False}"
+EQ_SECRETS_FILE="${EQ_SECRETS_FILE:=/secrets/secrets.yml}"
+EQ_KEYS_FILE="${EQ_KEYS_FILE:=/keys/keys.yml}"
+EQ_STORAGE_BACKEND="${EQ_STORAGE_BACKEND:=datastore}"
+EQ_ENABLE_SECURE_SESSION_COOKIE="${EQ_ENABLE_SECURE_SESSION_COOKIE:=True}"
+EQ_RABBITMQ_ENABLED="${EQ_RABBITMQ_ENABLED:=False}"
+EQ_ENABLE_HTML_MINIFY="${EQ_ENABLE_HTML_MINIFY:=False}"
+EQ_RABBITMQ_HOST="${EQ_RABBITMQ_HOST:=rabbit}"
+EQ_RABBITMQ_HOST_SECONDARY="${EQ_RABBITMQ_HOST_SECONDARY:=rabbit}"
+EQ_QUESTIONNAIRE_STATE_TABLE_NAME="${EQ_QUESTIONNAIRE_STATE_TABLE_NAME:=questionnaire-state}"
+EQ_SESSION_TABLE_NAME="${EQ_SESSION_TABLE_NAME:=eq-session}"
+EQ_USED_JTI_CLAIM_TABLE_NAME="${EQ_USED_JTI_CLAIM_TABLE_NAME:=used-jti-claim}"
+EQ_SUBMISSION_BACKEND="${EQ_SUBMISSION_BACKEND:=gcs}"
+EQ_FEEDBACK_BACKEND="${EQ_FEEDBACK_BACKEND:=gcs}"
+EQ_PUBLISHER_BACKEND="${EQ_PUBLISHER_BACKEND:=pubsub}"
+EQ_FULFILMENT_TOPIC_ID="${EQ_FULFILMENT_TOPIC_ID:=eq-fulfilment-topic}"
+EQ_INDIVIDUAL_RESPONSE_LIMIT="${EQ_INDIVIDUAL_RESPONSE_LIMIT:=75}"
+EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE="${EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE:=2021-04-28T02:00:00+00:00}"
+EQ_FEEDBACK_LIMIT="${EQ_FEEDBACK_LIMIT:=10}"
+HTTP_KEEP_ALIVE="${HTTP_KEEP_ALIVE:=650}"
+EQ_GOOGLE_TAG_MANAGER_ID="${EQ_GOOGLE_TAG_MANAGER_ID:=}"
+EQ_GOOGLE_TAG_MANAGER_AUTH="${EQ_GOOGLE_TAG_MANAGER_AUTH:=}"
+CDN_ASSETS_PATH="${CDN_ASSETS_PATH:=/design-system}"
+CONFIRMATION_EMAIL_LIMIT="${CONFIRMATION_EMAIL_LIMIT:=10}"
+EQ_SUBMISSION_CONFIRMATION_BACKEND="${EQ_SUBMISSION_CONFIRMATION_BACKEND:=cloud-tasks}"
```

Comment:
I think it would be clearer if all of the `_BACKEND` variables were grouped together.

Reply:
Grouped them together.